project_planning/wizard/reset_planning.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class WizardResetPlanning(models.TransientModel):
+    _name = 'wizard.reset.planning'
+
+    @api.multi
+    def reset_planning(self):
+        tasks = self.env['project.task'].search([])
+        tasks.write({
+            'week_1': 0,
+            'week_2': 0,
+            'week_3': 0,
+            'week_4': 0,
+        })
+        return True
```

Comment:
@Rona111 Be careful, there's still a pending PR for the reset planning wizard in GitLab. Please update the code with the one from the PR :)